src/service/performance-impl.js

```diff
@@ -221,7 +221,7 @@ export class Performance {
    * @private
    */
   setFlushParams_(params) {
-    this.viewer_.setFlushParams(params);
+    this.viewer_.sendMessageCancelUnsent('setFlushParams', params, false);
```

Comment:
Might be worth doing a quick math. But I think most important calls actually do wait on response. We had a couple of annoying errors related to this. E.g.

```
return promise1.then(() => {
  return viewer.sendMessage('message', payload);
}).then(result => {
});
```

The expectation here is that it'll block on the message response. In actuality this block will not block and instead immediately yield.

This error was so bad that I made third argument to be required. I wonder if we should just split methods to `sendMessage` and `sendMessageNoResult`. This would remove the ugly `Promise|undefined` response type and make types more complete.

/cc @lannka